src/AddTodoForm.js

```diff
@@ -1,24 +1,28 @@
-import React from "react"
+import React, { useState } from "react"
 
+function AddTodoForm(AddTodoForm) {
```

Comment:
please, revise this component - it is not functional

your component can receive props and you can use it like props.propName etc